src/module/toDoDom.js

```diff
@@ -1,58 +1,66 @@
-import Data from './data.js';
-
-export default class toDoDom {
-  static #datacheck = (done) => {
-    if (done) {
-      return 'checked';
-    }
-    return '';
-  }
-
-  static #dataDone = (done) => {
-    if (done) {
-      return 'text-dec';
-    }
-    return '';
-  }
-
-  static #addData = (data, container) => {
-    const listHtml = `
-    <li class="main-body-li" id="${data.id}" data-id="${data.id}">
-      <div class="width">
-        <span><input type="checkbox" name="list-check" class="list-check" ${toDoDom.#datacheck(data.done)}></span>
-        <span class="main-body-text"><input type="text" value="${data.text}" class="data-text ${toDoDom.#dataDone(data.done)}" contenteditable="true"></span>
-      </div>
-      <i class="fa-solid fa-square-minus remove-btn"></i>
-    </li>
-    `;
-
-    container.innerHTML += listHtml;
-  };
-
-  static #dataDownload = (data) => {
-    const datas = Data.dataGet();
-    datas.push(data);
-    Data.dataAdd(datas);
-  }
-
-  static data = (texts, container) => {
-    const task = {
-      text: texts,
-      done: false,
-      index: Data.dataGet().length + 1,
-      id: Date.now(),
-    };
-
-    if (texts !== '') {
-      toDoDom.#addData(task, container);
-      toDoDom.#dataDownload(task);
-    }
-  }
-
-  static datas = (container) => {
-    const datas = Data.dataGet();
-    datas.forEach((data) => {
-      toDoDom.#addData(data, container);
-    });
-  }
-}
\ No newline at end of file
+/**
+ * @jest-environment jsdom
+ */
+ import toDoDom from './toDoDom.js';
+ import Data from './data.js';
+ import Remove from './remove.js';
+ import Modify from './modify.js';
+ import * as base from './base.js';
+ 
+ describe('Testing add and remove', () => {
+   test('Add function test', () => {
+     document.body.innerHTML = '<ul class="main-body-ul"></ul>';
+     const appList = document.querySelector('.main-body-ul');
+     toDoDom.data('task1', appList);
```

Comment:
- [ ] Nice working on your tests, Kindly check it throws the error in the screenshot below when I run the test, 
![image](https://user-images.githubusercontent.com/48371156/203989180-5b54b1b6-82b0-42ca-8ef9-91883579f937.png)